doc/rtd/explanation/boot.rst

```diff
@@ -7,31 +7,11 @@ To be able to provide the functionality that it does, ``cloud-init`` must be
 integrated into the boot in a fairly controlled way. There are five
 stages to boot:
 
-1. Generator
-2. Local
-3. Network
-4. Config
-5. Final
+1. Local
```

Comment:
This stage is systemd-only, and the information that was contained here only described how to disable cloud-init. Since disabling cloud-init got moved to the new howto page, I didn't see any reason to keep this information here, since what value does that provide to a user? The other bootstages need to be understood sometimes for a user to know how to configure cloud-init, since cloud-init is not declarative. The fact that a systemd generator is used on systemd distros seems more like implementation detail than  actionable information to me. Why do you think this stage is just as important as the others? I can see this maybe being worth storing in a devdoc, but I'm not sure how it makes sense in the explanation category.

Thoughts?

Reply:
> I did a quick search through our docs, and we mention ds-identify a few times, but we don't really ever explain what it is or what it is doing.

100% agreed. The only "real" documentation of `ds-identify` imo is the source code.... which really sucks.

> What if we left

>> When booting under systemd, a generator_ will run that determines if cloud-init.target should be included in the boot goals.`

I wonder if it wouldn't make more sense to have a dedicated explanation page that describes details specific to each init system, rather than interspersing systemd-specific details throughout the docs, but I'm fine with the suggested approach for now.